charts/spire/charts/spire-server/README.md

```diff
@@ -240,6 +240,9 @@ In order to run Tornjak with simple HTTP Connection only, make sure you don't cr
 | `controllerManager.identities.jwtTTL`                            | Indicates an upper-bound time-to-live for JWT SVIDs. If unset, the cluster default will be chosen.                                                                                                                                                                                                                                                                   | `""`                                                                                           |
 | `controllerManager.identities.admin`                             | Indicates any pod matched by this identity will be an admin. Use this with extreme care.                                                                                                                                                                                                                                                                             | `false`                                                                                        |
 | `controllerManager.identities.downstream`                        | Set if this spire instance is a root server and the workloads are downstream servers.                                                                                                                                                                                                                                                                                | `false`                                                                                        |
+| `controllerManager.extraCRs.clusterSPIFFEIDs`                    | Specify additional ClusterSPIFFEID objects.                                                                                                                                                                                                                                                                                                                          | `{}`                                                                                           |
```

Comment:
What do you think of making the existing `identities` an array  instead of creating a new extraCRs section for it?

Reply:
Hmm...

If it was an array, the user couldn't update just one field due to the way helm overwrites arrays provided by the user. We could do it as a dictionary though.

So, something like:
```
identities:
  default:
    enabled: true
    spiffeIDTemplate: spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}
    podSelector: {}
    namespaceSelector: {}
    dnsNameTemplates: []
    federatesWith: []
    workloadSelectorTemplates: []
    ttl: ""
    jwtTTL: ""
    admin: false
    downstream: false
  other:
    same-stuff: here
```
The code would be the same as extraCRs.clusterSPIFFEIDs just pointed at identities instead and we can get rid of the extra code to handle the default.

What would we want to do with the other CR's in that case? Should we make another nesting level under identities for it? :
```
identities:
  dynamic:
    default:
      ...
  static:
    foo:
      ...
  federated:
    foo:
      ...
```

Or hang that stuff somewhere outside of identities?

Another option would be to structure things the cleanest way that doesn't break backwards comparability and handle it in the future with https://github.com/spiffe/helm-charts-hardened/issues/76

Reply:
Talked at the meeting. Use the crd name directly under identities. backwards compat breakage of moving things under default is ok.

Reply:
Feedback incorporated. I think this is ready now.